documentation/ipi-install/modules/ipi-install-creating-the-openshift-manifests.adoc

```diff
@@ -21,8 +22,8 @@ WARNING Discarding the Openshift Manifest that was provided in the target direct
 ifeval::[{release} <= 4.3]
 . Copy the `metal3-config.yaml` file to the `clusterconfigs/openshift` directory.
 +
-[source,bash]
+[source,terminal]
 ----
-# cp ~/metal3-config.yaml clusterconfigs/openshift/99_metal3-config.yaml
+[kni@provisioner ~]$ sudo cp ~/metal3-config.yaml clusterconfigs/openshift/99_metal3-config.yaml
```

Comment:
why does this need sudo?

Reply:
I think they did that for consistency, running as KNI vs running as Root, it's required